src/tools/web/README.md

```diff
@@ -12,7 +12,7 @@ Elektra-web requires:
 
 - [Elektra](https://libelektra.org/) with the [`yajl` plugin](https://master.libelektra.org/src/plugins/yajl/) installed
 - A recent [node.js](https://nodejs.org/en/) installation (at least 6.x)
-- [Go](https://golang.org/) with version > 1.11
+- [Go](https://golang.org/) with version > 1.13
```

Comment:
Why do we need this? This would make web not working on Debian stable.

Reply:
because this way we don't have any go dependencies at all.

Go compiles to a static binary, so that shouldn't be a problem. You only need go >= 1.13 to compile elektrad.

Reply:
Ahh, ok, is this about the "error" module? Yes, it makes sense to not have any deps. For Debian stable we need then some backport Go to compile.

Reply:
Yes it is!

Reply:
golang 1.13 is available in the official buster backports.